bin/pycbc_submit_dax

```diff
@@ -62,9 +62,20 @@ while true ; do
         "") shift 2 ;;
         *) CACHE_FILE=1
            export IFS=","
+           set +e
            for URL in $2; do
                curl --fail "$URL" >> _reuse.cache
+               #If curl fails but the file exists, write to cache, else exit
+               if [ $? != 0 ]; then
+                   if [ -f $URL ]; then
+                       cat $URL >> _reuse.cache
```

Comment:
Currently curl complains and writes an exit code to the screen if you have a relative path. Pycbc_submit_dax will still append to the cache if it can find the file, but it doesn't tell you that it found that file. It might be better to the user if I add, `echo "$URL cache file found. Appending..."`